shoplist/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="check_main">Check out your shopping lists!</string>
```

Comment:
adding resources great for any future translation and changes would be easier than the hard coded